src/testUtils/matchers.ts

```diff
@@ -11,6 +11,27 @@ interface MatchResult {
   pass: boolean;
 }
 
+function toBeUnique<T>(this: jest.MatcherUtils, received: any[]): MatchResult {
+  const seen: { [value: string]: boolean } = {};
+
+  for (const item of received) {
+    const stringified = JSON.stringify(item, null, 1);
+    if (seen[stringified]) {
+      return {
+        message: () => `Duplicate item in array:\n${stringified}`,
+        pass: false
+      };
+    } else {
+      seen[stringified] = true;
+    }
+  }
+
+  return {
+    message: () => 'Array is unique',
+    pass: true
+  };
+}
```

Comment:
Handy new jest matcher checking that the stringified value of each object in an array is unique